3-jasmine/sample.spec.ts

```diff
@@ -1,9 +1,15 @@
 import "jasmine";
-import {GeneratePESEL} from "../1-typescript-intro/pesel";
+import { GeneratePESEL } from "../1-typescript-intro/sample-5-modules-1";
+
 describe("Check if generator works coretly", () => {
-    xit("should work", () => {
-       
-    });
+  it("Check length of pesel and check data is on right place", () => {
+    let checkDate: string = "79521";
+    let b: Date = new Date("1979-05-21");
+    const generatePesel = GeneratePESEL(b);
+    console.log(generatePesel);
+    let checkDateExists: boolean = checkDate === generatePesel.slice(0, 5);
+    let checkLength: boolean = generatePesel.length === 10;
+    expect(checkLength && checkDateExists).toBe(true);
```

Comment:
In general, this test will check indeed if the function is working as expected. But to be more precise you could make two different tests for it. 

1) one would check if you have appropriate length of the string 
2) second would check if you have appropriate order of the numbers in the string but after you provide date in other format as input